### PR TITLE
Added 1.2 release notes.

### PR DESCRIPTION
**Pull request by AWSHurneyt**
### Description
Added 1.2 release notes.
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-alerting-dashboards-plugin.release-notes-1.2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Version 1.2.0.0 2021-11-04
+
+Compatible with OpenSearch Dashboards 1.2.0
+
+### Maintenance
+* Bumps version to 1.2 ([#128](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/128))
+* Cherry-pick commits from main branch to 1.x branch ([#131](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/131))
```
Comment (lezzago) [inline release-notes/opensearch-alerting-dashboards-plugin.release-notes-1.2.0.0.md:7]: This was not new changes for 1.2. Just a cherry pick from main to 1.x for backporting

File: release-notes/opensearch-alerting-dashboards-plugin.release-notes-1.2.0.0.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+* Added 1.2 release notes. ([#141](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/141))
+
+### Bug Fixes
+* Fixed a bug that displayed all alerts for a monitor on individual triggers' flyouts. Fixed a bug that displayed incorrect source for the condition field on the alerts flyout. Fixed a bug that displayed incorrect severity on the alerts flyout. Fixed a bug that prevented selecting query-level monitor alerts 1 by 1. Fixed bug relating to validation of popovers when defining monitor queries. ([#123](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/123))
```
Comment (lezzago) [inline release-notes/opensearch-alerting-dashboards-plugin.release-notes-1.2.0.0.md:11]: Isn't this a 1.1 change?

File: release-notes/opensearch-alerting-dashboards-plugin.release-notes-1.2.0.0.md (hunk 1 above)
Comment (AWSHurneyt) [reply release-notes/opensearch-alerting-dashboards-plugin.release-notes-1.2.0.0.md:7]: Will remove this, thank you!

File: release-notes/opensearch-alerting-dashboards-plugin.release-notes-1.2.0.0.md (hunk 2 above)
Comment (AWSHurneyt) [reply release-notes/opensearch-alerting-dashboards-plugin.release-notes-1.2.0.0.md:11]: Ah, you're right. I'll remove this; thank you!

**Review by lezzago — APPROVED** [review]

**Pull request merged**